packages/block-library/src/post-comments/index.php

```diff
@@ -15,17 +15,20 @@ function render_block_core_post_comments() {
 	if ( ! $post ) {
 		return '';
 	}
-	$comments = get_comments(
-		array(
-			'post_id' => $post->ID,
-		)
-	);
-	$output   = '';
-	// TODO: Handle nested comments.
-	foreach ( $comments as $comment ) {
-		$output .= '<p>' . $comment->comment_author . '<br />' . $comment->comment_content . '</p>';
-	}
-	return $output;
+
+	ob_start();
+	// This generates a deprecate message.
+	// Ideally this deprecation is removed.
+	comments_template( ABSPATH . WPINC . '/theme-compat/comments.php' );
```

Comment:
I think we shouldn't pass parameters here and let the `$file` default. Based on [the code here](https://github.com/WordPress/WordPress/blob/5308f570cb81edc29a1247ef4e816749a18ef6e2/wp-includes/comment-template.php#L1517) this absolute path will get appended to `STYLESHEETPATH` which will fail the `file_exists` check and default to what we are trying to pass anyway.

Also this allows us to use the current theme's comment template, which seems like a sensible fallback until we have the comments block suite?

Reply:
@vindl I think I agree with this idea. 

But it's interesting (in general) to think about pulling block output from a theme's PHP files though. Would this be the first time that takes place? Is it possible that pulling from a person's theme could create unrealistic expectations or sudden output breakages when this does change to use a comments blocks suite?

Reply:
I added the path as an attempt to "silence" the deprecated message but it didn't work. I'll remove.
I don't mind the theme comments fallback.

Reply:
> Is it possible that pulling from a person's theme could create unrealistic expectations or sudden output breakages when this does change to use a comments blocks suite?

I'm hoping that we'll be able to detect the usage of comments block suite, and only render this as a fallback when it's not present.